tests/standard_to_enhanced/navjoy_translator_data.py

```diff
@@ -0,0 +1,155 @@
+test_parse_reduction_zone_linestring_standard = {
```

Comment:
Should this be in data directory?

Reply:
Just moved all test data to the data directory